{
  "exploitdb": [],
  "itw": [
    {
      "id": "CVE-2021-29617",
      "description": "TensorFlow is an end-to-end open source platform for machine learning. An attacker can cause a denial of service via `CHECK`-fail in `tf.strings.substr` with invalid arguments. The fix will be included in TensorFlow 2.5.0. We will also cherrypick this commit on TensorFlow 2.4.2, TensorFlow 2.3.3, TensorFlow 2.2.3 and TensorFlow 2.1.4, as these are also affected and still in supported range.",
      "referenceURL": "https://nvd.nist.gov/vuln/detail/CVE-2021-29617",
      "reportURL": "https://github.com/tensorflow/tensorflow/security/advisories/GHSA-mmq6-q8r3-48fm",
      "timeStamp": "2021-07-27T17:12:00Z"
    }
  ],
  "trickest": [],
  "nomisec": [],
  "nuclei": [],
  "metasploit": [],
  "holloways": [],
  "openpoc": [
    {
      "cve": "CVE-2021-29617",
      "url": "https://github.com/tensorflow/tensorflow/security/advisories/GHSA-mmq6-q8r3-48fm",
      "added_at": "2021-07-27T17:12:00Z",
      "score": 0.4,
      "template_for": ""
    }
  ]
}
